packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -1029,7 +1044,7 @@ const ruleset: any = {
       description: "All operations should have a default (error) response.",
       message: "Operation is missing a default response.",
       severity: "error",
-      given: "$.paths.*.*.responses",
+      given: "$.paths.*.*.responses.*~",
```

Comment:
What is this change for? Added by mistake?

Reply:
just to be more explicit, means the same